pontoon/settings/base.py

```diff
@@ -191,6 +191,7 @@ def _site_url():
 TEMPLATES = [
     {
         'BACKEND': 'django_jinja.backend.Jinja2',
+        'NAME': 'jinja2',
```

Comment:
Just curious, what do we need that for? https://github.com/mozilla/pontoon/pull/639/files#diff-f94a39e4ed5b96101a481a61122332dcR194